tables.sql

```diff
@@ -0,0 +1,19 @@
+CREATE TABLE `release`
+(
+    `id` INTEGER NOT NULL AUTO_INCREMENT,
+    `name` VARCHAR(255) NOT NULL,
+    `info` VARCHAR(255) NOT NULL,
+    PRIMARY KEY (`id`)
+) ENGINE=InnoDB;
```

Comment:
oh and for the collation i've been using utf8mb4_unicode_ci - there's a newer one,  utf8mb4_unicode_520_ci, but it's only supported by recent versions of mysql which caused problems for us previously